client/src/actions/index.js

```diff
@@ -41,6 +41,17 @@ async function configFetch(dispatch) {
   });
 }
 
+async function userInfoFetch(dispatch) {
+  return fetchJson("userinfo").then((response) => {
```

Comment:
cc @bmccandless

Reply:
/userinfo is the standard name for this sort of thing.  It's used by the OpenID connect specification.

Reply:
Ah thanks @bmccandless! Given that they use `UserInfo` for their object name should we reflect the same naming style when not referencing the endpoint? https://openid.net/specs/openid-connect-core-1_0.html#UserInfo

Reply:
@seve @colinmegill, I don't have an opinion on the naming style;  I'll leave it up to you, maybe use whatever coding conventions you normally use.

Reply:
I am comfortable/prefer lowercase, agree that is standard

Reply:
Twilio uses PascalCase for things like PhoneNumber

Reply:
Let's agree we won't do Pascal, since all our endpoints begin lowercase. I've looked through all of them and it seems like this is the only, or one of the only, that is multiWord. @bmccandless if you are ok adopting camelCase, let's do that.

Reply:
@colinmegill The proposition is that the endpoint remains `userinfo` but once it is in the front end's hands we will adopt our standard of a camelCase `userInfo`.

Reply:
@colinmegill I'd like endpoint to be /userinfo;  no preference on how you name it in the front end code.

Reply:
@bmccandless let's leave the endpoint as is, I don't mind either way